cwltool/provenance.py

```diff
@@ -324,13 +322,13 @@ def __init__(self,
         self.user_provenance = user_provenance
         self.engine_uuid = research_object.engine_uuid
         self.add_to_manifest = self.research_object.add_to_manifest
-        if self.orcid:
+        if self.orcid is not None:
```

Comment:
empty orcid string is not valid and this would cause exception later on.

Reply:
Now we're only checking for the empty string instead of None